code/modules/mob/living/carbon/human/human_organs.dm

```diff
@@ -59,13 +59,20 @@
 	if(stance_damage < 0)
 		stance_damage = 0
 
+	if(stance_damage <= 6)
+		legless = FALSE
+
 	// standing is poor
 	if(stance_damage >= 8)
+		if(legless)
+			return
 		if(!(lying || resting))
+			legless = TRUE
 			if(!(NO_PAIN in dna.species.species_traits))
 				emote("scream")
 			custom_emote(1, "collapses!")
-		Weaken(5) //can't emote while weakened, apparently.
+		Weaken(5) //can't emote while weakened, apparently. Weakness kept here just to make the player collapse
```

Comment:
Remove this line and the one under it, you're basically weakening and immediately removing the weaken

Reply:
calling update_canmove() instead should do the trick

Reply:
The only other ways I can think of would be stunning or making them rest. What would you suggest?

Reply:
Use update_canmove() instead, if you can’t stand it’ll make you fall over